### ISSUE TITLE
Remove unix setup scripts + document dependencies instead

### ISSUE THREAD

**Issue by rminderhoud** [dev-osrose/osIROSE-new#173]
The instructions for setting up the project on linux are misleading. The script is not universal and will only work on Ubuntu based systems. Furthermore, it doesn't seem that it's actively maintained so it can cause some problems. My recommendation is that the script is removed and the dependencies are instead listed for the user to install on their own. Example:

```
Dependencies:
- G++7
- CMake 3.11+

Ubuntu:
` sudo apt-get install g++-7 unzip autoconf libtool libssl-dev python perl libncurses5-dev libreadline-dev`

Submodules:
`git submodule update --init --recursive`

Build:
mkdir build
cd build
cmake ..
ninja
```

Can't comment about the status of the powershell script as I have never used it.

If there's a desire to make a friendlier install path we can perhaps use docker to provide a consistent build environment.

**Comment by RavenX8** [comment]
We already provide a docker image for building.

https://github.com/dev-osrose/Dockerfiles

**Comment by hijxf** [comment]
Agreed. I think I might take a stab at making a Dockerfile for it as well as building on macOS.
